src/HealthChecks.UI.Data/HealthChecks.UI.Data.csproj

```diff
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
the bump was necessary, as EF Core not longer supports older TFMs

https://www.nuget.org/packages/Microsoft.EntityFrameworkCore#dependencies-body-tab

![image](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/assets/6011991/ae1cccdd-4e88-4371-ae39-1ff694ff35ef)